include/sparrow/layout/list_layout/list_array.hpp

```diff
@@ -190,8 +192,19 @@ namespace sparrow
         list_array_impl(self_type&&) = default;
         list_array_impl& operator=(self_type&&) = default;
 
+        template<class ... ARGS>
+        requires(mpl::excludes_copy_and_move_ctor_v<list_array_impl<BIG>, ARGS...>)
+        list_array_impl(ARGS && ... args): self_type(create_proxy(std::forward<ARGS>(args)...))
+        {}
```

Comment:
Do we want to allow constructions such as:
```cpp
list_array arr = {{1, 3, 4}, {2, 4}, {5, 1 ,9}};
```
?

Reply:
I don't think so. They are very special in the sense that they only work if the child array is a primitive array.

Reply:
on the other hand it might be a common use case...
I am unsure

Reply:
Ok let's see if users ask for it in the future.